README.md

```diff
@@ -29,6 +29,19 @@ Out of the box support following features:
 9. JWT based authentication.
 10. Tests for endpoints.
 
+## Prerequisites
+
+#### Google authorization
+
+Fill data from [googleAccount](https://console.developers.google.com/apis/) in `src/config/environment/index.js`
+```
+googleAccount: {
```

Comment:
Please rename 'googleAccount' to just 'google' in config (following convention with out services config, like 'mailgun')